stream-chat-android-ui-components/src/main/res/values/attrs_message_list_view.xml

```diff
@@ -27,6 +27,7 @@
         <attr name="streamUiMessageOptionIconColor" format="color" />
         <attr name="streamUiCopyMessageActionEnabled" format="color" />
         <attr name="streamUiDeleteConfirmationEnabled" format="boolean" />
+        <attr name="streamUiEditMessageEnabled" format="boolean" />
```

Comment:
I would rather call it `streamUiEditMessageOptionEnabled `, but one more time - we should decide and align other options